styles/modules/modals/_about.scss

```diff
@@ -17,18 +17,26 @@
     background: black;
   } 
 
-  .columns {
-    column-gap: $padMd;
-    max-height: fit-content;
+  .contributorList li {
+    padding-left: $padLg;
+    padding-bottom: $padSm;
+  }
+
+  #donationAddress {
```

Comment:
This should be done by putting a clrT2 class on the span. We set all colors using classes on the elements, so they can be changed by changing the color classes (that's going to be important later when we add theming).